pkg/job_controller/job.go

```diff
@@ -251,6 +251,11 @@ func (jc *JobController) ReconcileJobs(
 		return result, err
 	}
 
+	// Metring first pod launch delay when job state transit from created to running.
+	if !commonutil.IsRunning(*oldStatus) && commonutil.IsRunning(jobStatus) {
+		jc.Metrics.FirstPodLaunchDelaySeconds(activePods, metaObject, jobStatus)
+	}
+
 	jc.Metrics.AllPodsLaunchDelaySeconds(pods, metaObject, jobStatus)
```

Comment:
looks like this AllPodsLaunchDelaySeconds will be called again and again,  we only need it for the first All pods start running

Reply:
done, check before-after number of active pods.